src/CommandFormTypeMap.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Clearcode\CommandBusConsole;
+
+use SimpleBus\Message\CallableResolver\CallableResolver;
+use SimpleBus\Message\CallableResolver\Exception\UndefinedCallable;
+
+class CommandFormTypeMap
+{
+    /**
+     * @var array
+     */
+    private $commandFormTypes = [];
+
+    /**
+     * @var CallableResolver
+     */
+    private $callableResolver;
+
+
+    public function __construct(CallableResolver $callableResolver)
+    {
+        $this->callableResolver = $callableResolver;
+    }
+
+    /**
+     * @param array $commandFormTypes
+     */
+    public function processFormTypeServices($commandFormTypes)
+    {
+        $this->commandFormTypes = $commandFormTypes;
+    }
+
+    /**
+     * @param string $commandClass
+     * @return callable
+     */
+    public function get($commandClass)
```

Comment:
This class responisibility was to map from command class to form type class.
What are you returning here?